praw/objector.py

```diff
@@ -1,13 +1,19 @@
 """Provides the Objector class."""
+from .models.reddit.base import RedditBase
 from .exceptions import APIException, ClientException
 from .util import snake_case_keys
+from typing import Dict, List, Any, Union, Optional, NoReturn, TypeVar
```

Comment:
Can you move system-level imports above package-level ones?

Reply:
ok